action.yml

```diff
@@ -8,6 +8,10 @@ inputs:
     description: "The node version to use"
     required: false
     default: "20"
+  node-version-file:
+    description: "You may specify file path where node version is defined (e.g. package.json, .nvmrc, .node-version, .tool-versions.)"
+    required: false
+    default: ""
```

Comment:
Description is based on [that of `actions/setup-node@v4`](https://github.com/actions/setup-node/blob/c2ac33f2c62f978d6c944d9648125a294e56dc0b/action.yml#L10-L11) and `package-manager` option.